app/jobs/unzip_job.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require 'zip'
+
+# Change work version from a zipfile type to a browser type by unzipping the attached zip file.
+class UnzipJob < BaseDepositJob
+  queue_as :default
+
+  def perform(work_version)
+    zip_attached_file = work_version.attached_files.last
```

Comment:
There are lots of related DB operations in `#perform`. Will things get in a weird state without a transaction in place?

Reply:
Possibly, but since unzipping might take several minutes I'm reluctant to proactively transactionalize.